library/src/test/java/io/constructor/core/ConstructorIoIntegrationTest.kt

```diff
@@ -291,6 +288,110 @@ class ConstructorIoIntegrationTest {
         Thread.sleep(timeBetweenTests)
     }
 
+    @Test
+    fun getBrowseItemsResultsCRTAgainstRealResponse() {
+        runBlocking {
+            val browseResults = constructorIo.getBrowseItemsResultsCRT(listOf("dai_pid_2003438"))
```

Comment:
I don't think `dai_pid_2003438` is an item that exists in the `ZqXaOfXuBWD4s3XzCI1q` index. There are some tests that are failing because of that. We might want to use `10001` (Item 1) or another item from the index.